content/applications/inventory_and_mrp/manufacturing/workflows/allocation.rst

```diff
@@ -0,0 +1,104 @@
+
+.. image:: allocation/can-be-sold.png
+   :align: center
+   :alt: The Can Be Sold checkbox on a product form.
+
+Allocate products
+=================
+
+To allocate products or components from the |MO| in which they were produced to an |SO| or different
+|MO|, begin by navigating to :menuselection:`Manufacturing --> Orders --> Manufacturing Orders`.
+Click :guilabel:`New` to create a new |MO|.
+
+On the |MO|, select a product in the :guilabel:`Product` field, and specify the quantity to be
+produced in the :guilabel:`Quantity` field. Finally, click :guilabel:`Confirm` to confirm the |MO|.
+
+The rest of the allocation workflow depends on the current on-hand quantity of the product being
+manufactured, and whether or not there are any |SOs| or |MOs| which require the product, but have
+not already been allocated units of it.
+
+If there **ARE** existing |SOs| and |MOs| that require the product, **AND** there are too few units
+of the product on-hand to fulfill those orders, then a :icon:`fa-list` :guilabel:`Allocation` smart
+button appears at the top of the page as soon as the |MO| is confirmed.
+
+If there **ARE** existing |SOs| and |MOs| that require the product, **AND** there **ARE** enough
+units of the product on-hand to fulfill those orders, then the :icon:`fa-list`
+:guilabel:`Allocation` smart button only appears at the top of the page once the |MO| has been
+marked as done, by clicking :guilabel:`Produce All`.
+
+.. image:: allocation/allocation-button.png
+   :align: center
+   :alt: The Allocation smart button at the top of an MO.
+
+.. note::
+   If there **ARE NOT** any existing |SOs| and |MOs| that require the product, the :icon:`fa-list`
+   :guilabel:`Allocation` smart button does not appear, even when the |MO| has been marked as done.
+
+Click the :icon:`fa-list` :guilabel:`Allocation` smart button to open the *MRP Reception Report* for
+the |MO|.
+
+If a finished product is manufactured, this page lists any open customer delivery orders for which
+quantities of the product have yet to be reserved.
+
+If a manufacturing component is manufactured, this page lists any open |MOs| for which quantities of
+the component have yet to be reserved.
+
+Click the :guilabel:`Assign All` button to the right of a specific order to assign products for each
+quantity required to fulfill that order. For example, if an order requires one quantity of four
+units of the product, and one quantity of one unit of the product, clicking :guilabel:`Assign All`
+assigns five units of the product to fulfill both quantities.
+
+Alternatively, click :guilabel:`Assign` next to a specific quantity to only assign products to that
+quantity, and not any others in the order. Using the previous example, clicking :guilabel:`Assign`
+next to the quantity of one unit assigns a product to that quantity, but leaves the quantity of four
+units without any products assigned.
```

Comment:
I found this section a little confusing to me. I think having an example admonition where there is a clear example might be helpful. I know it would be for me =)